snippets/oadp-virtualization-supported-backup-types.adoc

```diff
@@ -0,0 +1,22 @@
+// Text snippet included in the following assemblies:
+//
+// ../backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
```

Comment:
```suggestion
// * <full-file-path>/backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
```

Reply:
Instead of the `..` you should include the full filepath, and add a * per the https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#text-snippet-file-metadata

Reply:
Except adding snippets in OCP docs does not work consistently. 

So we are probably going to stop sing snippets as there is no guarantee they will work